### PR TITLE
feat(crux-ui): separate status page from page 500

### PR DESCRIPTION
**Pull request by m8vago**
The service statuses from page 500 got extended and moved to /status
The new status page has version info now.
There are 4 services:
- Application UI
- API
- Authentication
- Database

### CODE REVIEW

File: web/crux-ui/src/server/kratos.ts [hunk 1]
```diff
@@ -8,25 +8,40 @@ import { missingParameter } from './error-middleware'
 
 const config = new Configuration({ basePath: process.env.KRATOS_URL })
 const kratos = new V0alpha2Api(config)
+const meta = new MetadataApi({
+  basePath: 'http://172.17.0.1:9434',
```
Comment (robot9706) [inline web/crux-ui/src/server/kratos.ts:12]: Wouldn't be better to use an env variable?

File: images/proto/Dockerfile [hunk 2]
```diff
@@ -13,7 +13,7 @@ RUN mkdir -p "$GOPATH/{src,bin,cache}" && chmod -R 777 "$GOPATH"
 RUN apk add nodejs npm alpine-sdk go protoc protobuf-dev rsync docker git cosign
 
 RUN npm i -g \
-    ts-proto@1.96.0 \
+    ts-proto@1.121.5 \
```
Comment (robot9706) [inline images/proto/Dockerfile:16]: Would this version work with crux-ui?

File: web/crux-ui/src/server/kratos.ts (hunk 1 above)
Comment (m8vago) [reply web/crux-ui/src/server/kratos.ts:12]: uh nice catch ty. totally forgot that one

File: protobuf/proto/crux.proto [hunk 3]
```diff
@@ -935,3 +937,18 @@ message NotificationResponse {
...
+  UNAVAILABLE = 1;
+  DISRUPTED = 2;
+  OPERATIONAL = 3;
+}
+
+message HealthResponse {
+  ServiceStatus status = 100;
+  string cruxVersion = 101;
+  optional string lastMigration = 102;
+}
```
Comment (polaroi8d) [inline protobuf/proto/crux.proto:954]: NW at EOF

File: web/crux/proto/crux.proto [hunk 4]
```diff
@@ -935,3 +937,18 @@ message NotificationResponse {
...
+  UNAVAILABLE = 1;
+  DISRUPTED = 2;
+  OPERATIONAL = 3;
+}
+
+message HealthResponse {
+  ServiceStatus status = 100;
+  string cruxVersion = 101;
+  optional string lastMigration = 102;
+}
```
Comment (polaroi8d) [inline web/crux/proto/crux.proto:954]: NW at EOF

File: web/crux/src/app/health/health.service.ts [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  private logger = new Logger(HealthService.name)
+
+  constructor(private prisma: PrismaService) {}
+
+  async getHealth(): Promise<HealthResponse> {
+    let lastMigration: string = null
+    try {
+      lastMigration = await this.prisma.findLastMigration()
+    } catch (err) {
+      const error: Error = err
```
Comment (polaroi8d) [inline web/crux/src/app/health/health.service.ts:16]: Why do you have to create an error object?

Comment (m8vago) [reply web/crux/src/app/health/health.service.ts:16]: cause typescript is a very fine language but apparently you can't give a type to a variable in the catch statement

**Review by robot9706 — APPROVED** [review]
😎

**Review by polaroi8d — APPROVED** [review]
Thanks, LGTM.

**Pull request merged**